coffeecard/CoffeeCard.Models/Entities/PosPurchase.cs

```diff
@@ -0,0 +1,13 @@
+namespace CoffeeCard.Models.Entities
+{
+    /// Represents a purchase issued by a barista at the point of sale in the cafe
+    public class PosPurhase
+    {
+        /// Primary key
+        public int Id { get; set; }
```

Comment:
Does the Id make sense, will it not always be the PurchaseId which is the primary id?